lib/dependencies/HarmonyImportDependencyParserPlugin.js

```diff
@@ -159,6 +159,7 @@ module.exports = class HarmonyImportDependencyParserPlugin {
 
 				const rootInfo = rightPart.rootInfo;
 				if (
+					typeof rootInfo === "string" ||
```

Comment:
can't get context, but hope it will not break our logic

Reply:
OH Whoops this is WRONG

Reply:
Oh nevermind I didn't see the `typeof`. Yes this is to handle the typecase: 
![image](https://user-images.githubusercontent.com/3408176/235236686-adf90eab-c315-4899-8956-38c421c48e8b.png)

Just a type gaurd for filtering out string types early. (Which the rest of the logic does but is technically invalid)